app/screens/DemoCreateProductScreen.tsx

```diff
@@ -0,0 +1,194 @@
+//     { label: 'Item 8', value: '8' },
+//   ];
+
+export const DemoCreateProductScreen: FC<DemoTabScreenProps<"DemoCreateProduct">> =
+  function DemoCreateProductScreen(_props) {
+    const [categories, setCategories] = useState([]);
+    useEffect(() => {
+      const fetchCategories = async () => {
+        try {
+          const response = await api.getCategories();
+          setCategories(response);
+        } catch (error) {
+          console.error("Error fetching categories:", error);
+        }
+      };
+
+      fetchCategories();
+    }, []); // Empty dependency array ensures the effect runs only once, on component mount
+
+
+    return (
+      <Screen preset="scroll" safeAreaEdges={["top"]} contentContainerStyle={$container}>
+        <View style={$createContainer}>
+          <Text style={$title}>Create product</Text>
+
+          <TextField
+            autoCapitalize="none"
+            containerStyle={$textField}
+            autoCorrect={false}
+            keyboardType="email-address"
+            label="Product Name"
+            placeholder="Enter product name"
+          />
+
+          <View style={$informationField}>
+            <TextField
+              autoCapitalize="none"
+              containerStyle={$informationFieldTextField}
+              autoCorrect={false}
+              keyboardType="email-address"
+              label="Price ($)"
+              placeholder="Ex: 100"
+            />
+
+            <TextField
+              autoCapitalize="none"
+              containerStyle={$informationFieldTextField}
+              autoCorrect={false}
+              keyboardType="email-address"
+              label="Discount (%)"
+              placeholder="Ex: 10"
+            />
+          </View>
+
+          <View style={$descriptionField}>
+            <TextField
+              style={$descriptionTextField}
+              label="Description"
+              placeholder="Enter description"
+              multiline
+            />
+          </View>
+
+          <Text style={$label}>Category</Text>
+          <DropdownComponent data={categories} placeholder="Select category"/>
+          <View style={$conditionField}>
+            <ControlledToggle variant="radio" label="New" />
+            <ControlledToggle variant="radio" label="Like new" />
+            <ControlledToggle variant="radio" label="Old" />
+          </View>
+
+          <View style={$uploadArea}>
+            <Text style={$uploadText}>Upload Image</Text>
+            <Button style={$uploadButton} onPress={openImagePicker}>
+              Upload
+            </Button>
+          </View>
+        </View>
+      </Screen>
+    )
+  }
+
+const $container: ViewStyle = {
+  paddingTop: spacing.lg,
+  paddingBottom: spacing.xxl,
+  paddingHorizontal: spacing.lg,
+  backgroundColor: 'white',
+}
+
+const $label: TextStyle = {
+  marginTop: spacing.lg,
+  display: "flex",
+  flexDirection: "row",
+  width: "100%",
+  fontWeight: "bold",
```

Comment:
dò lại những cái có thể config trong prop đi, đừng bỏ vào style nữa

Reply:
component nào thì css độc lập component nấy

Reply:
check lại component <Text></Text> nó có 1 props.size đó

Reply:
ok i saw it